packages/integration-tests/projects/suite-web/support/pageObjects/settingsCryptoObject.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference types="cypress" />
+
+class SettingsCrypto {
+    activateCoin(coin: string) {
```

Comment:
the same 
https://github.com/trezor/trezor-suite/blob/develop/suite-common/wallet-config/src/networksConfig.ts#L386
feel free to ignroe